src/client/ReservationComponents/Reservation.js

```diff
@@ -0,0 +1,94 @@
+import React from "react";
+import { useState } from "react";
+export function MakeReservation(props) {
+  const [mealId, setMealId] = useState("");
+  const [name, setName] = useState("");
+  const [phonenumber, setPhonenumber] = useState("");
+  const [email, setEmail] = useState("");
+
+  function handleSubmit(event) {
+    event.preventDefault();
+
+    const data = {
+      number_of_guests: 1,
+      meal_id: props.mealId,
+      created_date: "2022-04-15",
+      contact_name: name,
+      contact_phonenumber: phonenumber,
+      contact_email: email,
+    };
+
+    fetch("http://localhost:3000/api/reservations", {
+      method: "POST", // or 'PUT'
```

Comment:
Do you understand when to use POST and when to use PUT?
And how they are defined differently in your API? And what they mean in the sense of a REST API?